### PR TITLE
React Capstone

### PR DESCRIPTION
**Pull request by aron-net**
:heavy_check_mark: The project is a single-page application(SPA) built with react and redux.
:heavy_check_mark: The data retrieved from the API is stored in the redux store.
:heavy_check_mark: The data filter retrieve from the API using a stateless filter component.
:heavy_check_mark:  Every page has a unique route with the SPA.
:heavy_check_mark:  The project is deployed there is live dom
:heavy_check_mark: Record video presenting the features -
- [Recorded Video](https://www.loom.com/share/8eacf93aa8d6451393e34e219ff7adec)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +61,10 @@ Give a ⭐️ if you like this project!
...
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
+- [Microverse](https://www.microverse.org/go?gclid=Cj0KCQjwyOuYBhCGARIsAIdGQRONnNmf5hcDufHpehQFsgtsW3CaTjdlZl7Fa_rIDK1cyyEz4MRY754aAg_REALw_wcB/)
+- [ReactJS Organization](https://reactjs.org/) for the amazing [Documentation](https://reactjs.org/docs/getting-started.html/) on React.
+- [Redux Organization](https://redux.js.org/) for the amazing [Redux Essentials Tutorial](https://redux.js.org/tutorials/essentials/part-1-overview-concepts/) on React.
+- [Nelson Sakwa](https://www.behance.net/sakwadesignstudio/) for the amazing [on Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)/)
```
Comment (mateo951) [inline README.md:67]: Well done giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 🤓

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import Home from './Components/Home/Home';
+import NavBar from './Components/NavBar/NavBar';
+
+const App = () => (
+  <div className="App">
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/:name" element={<Countries />} />
+    </Routes>
```
Comment (mateo951) [inline src/App.js:12]: The project contains a `Home` page and a `Detail` page 🤓 Well done!

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Route, Routes } from 'react-router-dom';
```
Comment (mateo951) [inline src/App.js:1]: You successfully created a  single page application (SPA) using `react-router-dom` to serve each page without reloading 🙌🔥

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import Countries from './Components/Countries/Countries';
+import Home from './Components/Home/Home';
+import NavBar from './Components/NavBar/NavBar';
+
+const App = () => (
+  <div className="App">
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/:name" element={<Countries />} />
```
Comment (mateo951) [inline src/App.js:11]: The category item detail page has its unique route within the SPA 🚀

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>
```
Comment (mateo951) [inline src/index.js:17]: The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly 🤓

File: src/Components/Home/Home.css [hunk 6]
```diff
@@ -0,0 +1,155 @@
+* {
```
Comment (mateo951) [inline src/Components/Home/Home.css:1]: Even though I really like your design, it's important that we follow the given design as much as possible. It must be really close to being an exact replica 🤓

The design 
image

Please be sure to work on the layout of your design. For example, try mimicking the composition and space between elements.

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @aron-helu, 

**🔥 Highlights 🔥**
- General Requirements 💪🤓
- Great Video Presentation ✔
- App deployed using Github Pages 🚀
- Implementation of API 🤯

Excelente work! I had a great time reviewing your project 😎 There are some issues one more still needs work to go to the next project but you are almost there!

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mwanawabangona — APPROVED** [review]
Hi @aron-helu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**